packages/wdio-cli/src/launcher.js

```diff
@@ -55,49 +55,56 @@ class Launcher {
 
     /**
      * run sequence
-     * @return  {Promise} that only gets resolves with either an exitCode or an error
+     * @param   {Function}  callback    callback
+     * @return  {Promise}               that only gets resolves with either an exitCode or an error
      */
-    async run () {
-        let config = this.configParser.getConfig()
-        let caps = this.configParser.getCapabilities()
-        const launcher = initialiseServices(config, caps, 'launcher')
-
+    async run (callback) {
         /**
-         * run pre test tasks for runner plugins
-         * (e.g. deploy Lambda function to AWS)
+         * catches ctrl+c event
          */
-        await this.runner.initialise()
+        exitHook(:: this.exitHandler)
+        let exitCode
+        let onCompleteResults
 
-        /**
-         * run onPrepare hook
-         */
-        log.info('Run onPrepare hook')
-        await runOnPrepareHook(config.onPrepare, config, caps)
-        await runServiceHook(launcher, 'onPrepare', config, caps)
+        try {
+            let config = this.configParser.getConfig()
+            let caps = this.configParser.getCapabilities()
```

Comment:
caps and config can be const

Reply:
Yeah, I just copy pasted existing code, didn't want to improve it or modify it. 
Let me change it

Reply:
@CrispusDH fixed